tools/downloader/common.py

```diff
@@ -27,6 +27,7 @@
 import yaml
 
 DOWNLOAD_TIMEOUT = 5 * 60
+MODEL_ROOT = (Path(__file__).resolve().parent / '../../models').resolve()
```

Comment:
Path also have property `parents`, which return list of logical ancestors  of current path:
https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.parents
I think you can use it for getting access to 3 levels up path instead of `../../`

Reply:
please review again